### ISSUE TITLE
Stable

### ISSUE THREAD

**Issue by nonzzz** [nonzzz/tar#6]
## TL;DR

Before lib stable. I want to make sure that some of not support `ustar` format based usage cases. Like global pax and etc.

Now the lib support standard `ustar` field and a partial pax fields like `path`,`linkpath` and etc. But the current pax implement only support 'x' no 'g'. Full detail see [EXTENDED DESCRIPTION](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/pax.html#tag_20_92_13_06)

In my option, before stable this lib need support common extensions. (Note this lib doesn't want to implement too many features or support too many formats. It should be as simple as possible.)

## Others

Maybe should provide an fs integration?
